WordPress/Classes/ViewRelated/Stats/JetpackLoginViewController.swift

```diff
@@ -114,38 +118,43 @@ class JetpackLoginViewController: UIViewController {
         var message: String
 
         if jetPack.isConnected {
-            if jetPack.isUpdatedToRequiredVersion {
-                message = NSLocalizedString("Looks like you have Jetpack set up on your site. Congrats! " +
-                                            "Log in with your WordPress.com credentials to enable " +
-                                            "Stats and Notifications.",
-                                            comment: "Message asking the user to sign into Jetpack with WordPress.com credentials")
-            } else {
-                message = String.localizedStringWithFormat(NSLocalizedString("Jetpack %@ or later is required. " +
-                                                                             "Do you want to update Jetpack?",
-                                                                             comment: "Message stating the minimum required " +
-                                                                             "version for Jetpack and asks the user " +
-                                                                             "if they want to upgrade"), JetpackState.minimumVersionRequired)
-            }
+            message = jetPack.isUpdatedToRequiredVersion ? Constants.Jetpack.isUpdated : Constants.Jetpack.updateRequired
         } else {
-            switch promptType {
-            case .stats:
-                message = NSLocalizedString("To use Stats on your site, you'll need to install the Jetpack plugin.\n Would you like to set up Jetpack?",
-                                            comment: "Message asking the user if they want to set up Jetpack from stats")
-            case .notifications:
-                message = NSLocalizedString("To get helpful notifications on your phone from your WordPress site, you'll need to install the Jetpack plugin. Would you like to set up Jetpack?",
-                                            comment: "Message asking the user if they want to set up Jetpack from notifications")
-            }
+            message = promptType.message
         }
         descriptionLabel.text = message
         descriptionLabel.sizeToFit()
 
-        var title = NSLocalizedString("Set up Jetpack", comment: "Title of a button for Jetpack Installation.")
+        let title = Feature.enabled(.jetpackRemoteInstallation) ? Constants.Buttons.jetpackInstallTitle : Constants.Buttons.jetpackSetupTitle
         installJetpackButton.setTitle(title, for: .normal)
         installJetpackButton.isHidden = hasJetpack
 
-        title = NSLocalizedString("Log in", comment: "Title of a button for signing in.")
-        signinButton.setTitle(title, for: .normal)
+        signinButton.setTitle(Constants.Buttons.loginTitle, for: .normal)
         signinButton.isHidden = !hasJetpack
+
+        // If .jetpackRemoteInstallation is not enabled
```

Comment:
Just for cleanliness, we can probably remove these comments as they essentially describe exactly what the code beneath does :)